spm_tutorials/spm-FaceRep_smdl.json

```diff
@@ -0,0 +1,147 @@
+{
+    "BIDSModelVersion": "1.0.0",
+    "Name": "SPM_FaceRepetition",
+    "Description": "Look at a) main effect of seeing task, b) see if repetition affects BOLD response, c) effect of motion",
+    "Input": {
+            "task": "facerepetition"
+    },
+    "Nodes": [
+        {
+            "Level": "run",
+            "Type": "glm",
+            "Transformations": {
+                "Transformer": "pybids-transforms-v1",
+                "Instructions": [
+                    {
+                    "Name": "AND",
+                    "Input": [
+                        "face_type==famous",
+                        "repetition_type==1"
+                    ],
+                    "Output": [
+                        "famous_1"
+                    ]
+                    },
```

Comment:
this instruction and the following ones aims to create 4 variables from the 2X2 design of the dataset, but I have no idea if this is the correct way to do this

Reply:
I may probably be need help with figuring out which transformers to use to do this.

@effigies